src/time/format.cr

```diff
@@ -27,14 +30,15 @@
 # * **%j**: day of year, zero padded (001, 002, ..., 365)
 # * **%k**: hour of the day, 24-hour clock, blank padded (" 0", " 1", ..., "24")
 # * **%l**: hour of the day, 12-hour clock, blank padded (" 0", " 1", ..., "12")
-# * **%3N**, **%L**: milliseconds, zero padded (000, 001, ..., 999)
-# * **%6N**: microseconds, zero padded (000000, 000001, ..., 999999)
-# * **%9N**: nanoseconds, zero padded (000000000, 000000001, ..., 999999999)
-# * **%N**: second fraction, zero padded. (Same as `%9N` but may consume more than 9 digits while parsing)
+# * **%L**: milliseconds, zero padded (000, 001, ..., 999)
 # * **%m**: month number, zero padded (01, 02, ..., 12)
 # * **%_m**: month number, blank padded (" 1", " 2", ..., "12")
 # * **%-m**: month number (1, 2, ..., 12)
 # * **%M**: minute, zero padded (00, 01, 02, ..., 59)
+# * **%3N**, **%L**: milliseconds, zero padded (000, 001, ..., 999)
+# * **%6N**: microseconds, zero padded (000000, 000001, ..., 999999)
+# * **%9N**: nanoseconds, zero padded (000000000, 000000001, ..., 999999999)
+# * **%N**: second fraction, zero padded. (Same as `%9N` but may consume more than 9 digits while parsing)
```

Comment:
Why this doc change?

Reply:
It helps discoverability if the directives are lexically ordered. It's unrelated, though. If you like, I can make a separate PR for this.